rc/tools/git.kak

```diff
@@ -189,6 +196,7 @@ Available commands:\n  add\n  rm\n  blame\n  commit\n  checkout\n  diff\n  hide-
            ;;
        update-diff) update_diff ;;
        commit) shift; commit "$@" ;;
+       init) shift; init "$@" ;;
```

Comment:
could probably just have `shift; git init "$@" > /dev/null 2>&1` here instead of introducing a function.

Reply:
done